src/config/src/config.erl

```diff
@@ -612,7 +612,13 @@ to_float_test() ->
     ?assertEqual(1.0, to_float(<<"1.0">>)),
     ?assertEqual(1.0, to_float("1.0")),
     ?assertEqual(-1.1, to_float("-01.1")),
-    ?assertEqual(0.0, to_float("-0.0")),
+    ?assertEqual(
+        case erlang:system_info(otp_release) of
+            Ver when Ver =:= "27" -> -0.0;
+            _ -> 0.0
+        end,
+        to_float("-0.0")
+    ),
```

Comment:
For `-0.0` and `0.0` let's have two cases and use numerical comparison == instead of `?assertEqual/2` which probably does exact comparisons.

That is `?assertEqual(0.0 == to_float("-0.0"))` and `?assertEqual(0.0 == to_float("0.0"))`

Reply:
Changed to `?assertEquiv()`.

Reply:
Ooh, good find @jiahuili430. I had no idea we even had `?assertEquiv/1`It looks like was introduced in https://github.com/apache/couchdb/commit/306d2e2df54fa9406adf1022c4d250040b90e42a by Adam. Well that's perfect for what we want, let's use it!